### ISSUE TITLE
model continuous_datadriven_bin has a problem

### ISSUE THREAD

**Issue by buaaJeremyduan** [gy20073/BDD_Driving_Model#9]
Has anyone tried the model of continuous_datadriven_bin? I use the file model.ckpt-261001.bestmodel and the output is np.array([1, 362]). However, its argmax is always 87

**Comment by buaaJeremyduan** [comment]
I initialize the wrapper like this:
a = wrapper.Wrapper("continuous_datadriven_bin", 
             "/home/duan/git/BDD_Driving_Model/data/continuous_datadriven_bin/model.ckpt-261001.bestmodel", 1, "config", ".", True)
I then check the logits_v it outputs:
logits_v, self.state_value = self.sess.run([self.logits, self.state_tensor], feed_dict=fd)
it should be the future_angular_volocity, right?
Its argmax is alway 87

**Comment by s7ev3n** [comment]
@buaaJeremyduan I met the same problem, its argmax is always 87, have you solved it?

**Comment by merryHunter** [comment]
Hi, during evaluation of continuous_datadriven_bin on BDDV data I've come across this too. @gy20073 , could you comment on this issue, please? Also, do you have checkpoints for linear or log bin models that doesn't have this issue?
